.github/workflows/release.yml

```
@@ -33,7 +39,43 @@ jobs:
 
       - run: make -C .ci dist
 
-      - run: make -C .ci push-docker
+      - name: generate build provenance
+        uses: github-early-access/generate-build-provenance@main
+        with:
+          subject-path: "${{ github.workspace }}/build/aws/*.zip"
```

```suggestion
          subject-path: "${{ github.workspace }}/build/aws/elastic-apm-node-lambda-layer-*.zip"
```

Q: Is there much value in this attestation? We don't upload the .zip file as an artifact... other than to AWS regions as lambda layers. I suppose the attestation will be there if someone wants to download those layer zip files (https://stackoverflow.com/questions/56614037/how-to-download-aws-lambda-layer) and check them.

---

I decided to create an attestation for anything we publish, regardless of where those artifacts are stored. I think it's valuable we also do that for the lambda layers